python/lib/evaluations/pyproject.toml

```diff
@@ -0,0 +1,36 @@
+[build-system]
+requires = ["setuptools >= 70.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "dmod.evaluations"
+authors = [
+    { name = "Chris Tubbs" },
+    { name = "Austin Raney", email = "austin.raney@noaa.gov" },
+    { name = "Robert Bartel" },
+]
+dependencies = [
+    "dmod.core",
```

Comment:
This may not be the place to address it, but it is probably a bug that no version constraint for `dmod.core` is specified here (likely also true for the next line and `dmod.metrics`).